internal/server/data/data_test.go

```diff
@@ -15,13 +15,22 @@ import (
 	"github.com/infrahq/infra/uid"
 )
 
+// TODO: replace
 func setup(t *testing.T) *gorm.DB {
+	t.Helper()
 	driver, err := NewSQLiteDriver("file::memory:")
 	assert.NilError(t, err)
+	return setupDB(t, driver)
+}
 
-	db, err := NewDB(driver)
+func setupDB(t *testing.T, driver gorm.Dialector) *gorm.DB {
+	t.Helper()
+	db, err := NewRawDB(driver)
 	assert.NilError(t, err)
 
+	// TODO: mirror production
```

Comment:
Does this `TODO` mean you're going to pre-populate this with some production data?

Reply:
Ah, no this TODO is about making sure the test code paths match production.

We used to have a `NewDB` function that setup the database for production code paths. Then we introduced `NewRawDB` which production uses, but all of our tests use `NewDB`. This divergence means our tests can lie to us because they are not testing the right state of the db.

This TODO is about fixing that problem. I'll word it better.